src/www/style.css

```diff
@@ -14,20 +14,49 @@
  * limitations under the License.
  */
 
-html,
-body {
-  height: 100%;
-}
+:root {
+  /* tokens */
+  --min-supported-device-width: 320px;
 
-body {
-  background: #efefef;
-  margin: 0;
-  font-family: Roboto, sans-serif;
-}
+  --outline-primary: hsl(170, 60%, 46%);
+  --outline-error: hsl(4, 90%, 58%);
+
+  --outline-black: hsl(0, 0%, 0%);
+  --outline-off-black: hsl(0, 0%, 15%);
+  --outline-dark-gray: hsl(213, 5%, 39%);
+  --outline-medium-gray: hsl(0, 0%, 45%);
+  --outline-light-gray: hsl(0, 0%, 95%);
+  --outline-white: hsl(0, 0%, 100%);
+
+  --outline-font-family: Roboto, sans-serif;
 
-html {
-  -webkit-touch-callout: none; /* Safari mobile */
-  -webkit-user-select: none; /* Safari */
-  -ms-user-select: none; /* IE/Edge */
-  user-select: none; /* Chrome */
+  --outline-corner: 2px;
+
+  --outline-large-gutter: 2rem;
+  --outline-gutter: 1rem;
+  --outline-slim-gutter: 0.75rem;
+  --outline-mini-gutter: 0.25rem;
+
+  /* theme */
+  --outline-card-background: var(--outline-white);
+  --outline-card-footer: var(--outline-light-gray);
+  --outline-text-color: var(--outline-black);
+  --outline-label-color: var(--outline-medium-gray);
+  --outline-hairline: 1px solid hsl(0, 0%, 85%);
+  --outline-elevation: rgb(0 0 0 / 30%) 0px 2px 3px 2px;
+
+  --mdc-typography-font-family: var(--outline-font-family);
+  --mdc-theme-primary: var(--outline-primary);
 }
+
+/* TODO(daniellacosse): enable this when everything has been color scheme'd */
+/* @media (prefers-color-scheme: dark) {
```

Comment:
commenting this in actually enables dark mode if you have that preference set at your OS level. only problem is this only works for the code I've rewritten so far.